docs/writing-docs/docs-page.md

```diff
@@ -54,30 +54,33 @@ If you want to organize your documentation differently for component groups, we
 Replace the DocsPage template with your own to customize its contents.
 
 ### With null to remove docs
- 
+
 Override the `docs.page` [parameter](../writing-stories/parameters.md) with `null` to remove its contents.
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
-  paths={[
-    'common/button-story-disable-docspage-component.js.mdx',
-  ]}
+paths={[
+'common/button-story-disable-docspage-component.js.mdx',
+]}
```

Comment:
@RickCardoso if you could revert back to the previous spacing in the documentation I'd appreciate it. It's a matter of consistency and to avoid any issues with the documentation once it's built.

Reply:
Not intentional, for sure

Also not sure how to fix it in my code, it's already spaced, I'll try here in GitHub
![image](https://user-images.githubusercontent.com/15681333/126534577-38e0449d-ed7c-4df5-8f20-e3634b73857f.png)

Reply:
Tried to fix this and propably made it worse, not sure why the spacing is being changed